src/modules/base/Module.sol

```diff
@@ -97,23 +97,24 @@ abstract contract Module is
         bytes32 ownerRole = authorizer.getOwnerRole();
         bytes32 managerRole = authorizer.getManagerRole();
 
-        if (!authorizer.hasRole(managerRole, _msgSender())) {
-            revert Module__CallerNotAuthorized(managerRole, _msgSender());
-        } else if (!authorizer.hasRole(ownerRole, _msgSender())) {
-            revert Module__CallerNotAuthorized(ownerRole, _msgSender());
+        if (
+            authorizer.hasRole(ownerRole, _msgSender())
+                || authorizer.hasRole(managerRole, _msgSender())
+        ) {
+            _;
+        } else {
+            if (authorizer.hasRole(ownerRole, _msgSender())) {
```

Comment:
Isn't this missing a `!`? Otherwise, this will never be reached, as the if-clause in line 101/102 is already checking for it. Or am I missing something?

Reply:
Maybe to add to that, not sure if it is important with what kind of revert message it gets reverted, but the way it reverts now is reverse from how it was before, as well as the `revert Module__CallerNotAuthorized(ownerRole, _msgSender());` being the catch all revert message

Reply:
Yes to both. I think not being owner is a better catch-all message. Do you guys disagree?

Reply:
I think one error message is probably enough to signal that the issue of a failed tx is an authorization issue. So I would be fine with just reverting with the owner catch-all, personally.